core-api/src/main/java/com/optimizely/ab/config/Experiment.java

```diff
@@ -44,6 +45,7 @@
     private final String groupId;
 
     private final List<String> audienceIds;
```

Comment:
I am strongly against this idea.  There does not seem to be any logical reason to make a change such as this.

Reply:
This was my reasoning:

> I think this will allow us to
>
> 1. Reduce the amount of forking in SDK code, although this may be irrelevant given how simple `audienceIds` currently is.
> 2. Eventually consolidate on `audienceIds` once all installed SDKs know to treat it as a condition tree. This'd be nice since `audienceConditions` (or whatever other name we might have chosen for the new property) is more confusing.

We should discuss more broadly than in this PR if we want to change the proposed behavior